lib/compilers/pascal-win.js

```diff
@@ -126,7 +126,7 @@ export class PascalWinCompiler extends BaseCompiler {
 
         await fs.writeFile(inputFilename, source);
 
-        if (files) {
+        if (files && files.length > 0) {
```

Comment:
Ditto comment from above: not sure if this was something that fell out as being needed, or if it's specific to the external asm-parser?